packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -1137,6 +1137,31 @@ class AccountInternal extends PureComponent<
     });
   };
 
+  onUnmarkTransactionsAsImported = async (ids: string[]) => {
+    this.setState({ workingHard: true });
+
+    const { data } = await runQuery(
+      q('transactions')
+        .filter({ id: { $oneof: ids } })
+        .select('*')
+        .options({ splits: 'grouped' }),
+    );
+
+    const transactions: TransactionEntity[] = data;
+
+    if (!transactions || transactions.length === 0) {
+      return;
+    }
+
+    const importedTransactionIds = transactions
+      .filter(t => t.imported_payee)
+      .map(t => t.id);
+
+    this.onBatchEdit('imported_payee', importedTransactionIds);
+
+    this.refetchTransactions();
+  };
+
```

Comment:
@joel-jeremy will know better than me on this, is this okay here or should it be moved to packages/desktop-client/src/hooks/useTransactionBatchActions.ts?